src/jit/codegenarmarch.cpp

```diff
@@ -3838,4 +3838,140 @@ void CodeGen::genStructReturn(GenTree* treeNode)
 
     } // op1 must be multi-reg GT_CALL
 }
+
+//------------------------------------------------------------------------
+// genAllocLclFrame: Probe the stack and allocate the local stack frame: subtract from SP.
+//
+// Notes:
+//      On ARM64, this only does the probing; allocating the frame is done when
+//      callee-saved registers are saved.
+//
+void CodeGen::genAllocLclFrame(unsigned frameSize, regNumber initReg, bool* pInitRegZeroed, regMaskTP maskArgRegsLiveIn)
+{
+    assert(compiler->compGeneratingProlog);
+
+    if (frameSize == 0)
+    {
+        return;
+    }
+
+    const target_size_t pageSize = compiler->eeGetPageSize();
+
+    assert(!compiler->info.compPublishStubParam || (REG_SECRET_STUB_PARAM != initReg));
+
+    if (frameSize < pageSize)
+    {
+#ifdef _TARGET_ARM_
+        // Frame size is (0x0008..0x1000)
+        inst_RV_IV(INS_sub, REG_SPBASE, frameSize, EA_PTRSIZE);
+#endif // _TARGET_ARM_
+    }
+    else if (frameSize < compiler->getVeryLargeFrameSize())
+    {
+        // Frame size is (0x1000..0x3000)
+        CLANG_FORMAT_COMMENT_ANCHOR;
```

Comment:
Minor: There are at least to of these `CLANG_FORMAT_COMMENT_ANCHOR`s that can be removed now.

Reply:
Thanks. Removed all of them. jit-format shows no changes after that.